rct229/rulesets/ashrae9012019/section19/section19rule4.py

```diff
@@ -0,0 +1,321 @@
+
+                            most_used_weekday_hourly_schedule_b = (
+                                get_most_used_weekday_hourly_schedule(
+                                    multiplier_sch_hourly_value_b,
+                                    day_of_week_for_january_1,
+                                )
+                            )
+                            int_lgt_pass_cooling_b = (
+                                design_cooling_multiplier_sch_b
+                                == most_used_weekday_hourly_schedule_b
+                            )
+
+                    # check misc equipment
+                    misc_pass_cooling_b = False
+                    for misc_equip_b in space_b["miscellaneous_equipment"]:
+                        if not misc_pass_cooling_b:
+                            multiplier_sch_light_b = getattr_(
+                                misc_equip_b,
+                                "Miscellaneous Equipment",
+                                "multiplier_schedule",
+                            )
+                            multiplier_sch_hourly_value_b = schedule_b[
+                                multiplier_sch_light_b
+                            ]["hourly_values"]
+                            design_cooling_multiplier_sch_b = schedule_b[
+                                multiplier_sch_light_b
+                            ]["hourly_cooling_design_day"]
+
+                            most_used_weekday_hourly_schedule_b = (
+                                get_most_used_weekday_hourly_schedule(
+                                    multiplier_sch_hourly_value_b,
+                                    day_of_week_for_january_1,
+                                )
+                            )
+                            misc_pass_cooling_b = (
+                                design_cooling_multiplier_sch_b
+                                == most_used_weekday_hourly_schedule_b
+                            )
+
+                    return {
+                        "inf_pass_cooling_b": inf_pass_cooling_b,
+                        "occ_pass_cooling_b": occ_pass_cooling_b,
+                        "int_lgt_pass_cooling_b": int_lgt_pass_cooling_b,
+                        "misc_pass_cooling_b": misc_pass_cooling_b,
+                    }
+
+                def manual_check_required(self, context, calc_vals=None, data=None):
+                    space_b = context.baseline
+
+                    is_lighting_bldg_area_defined_b = data[
+                        "is_lighting_bldg_area_defined_b"
+                    ]
+                    is_building_area_MF_dormitory_or_hotel_b = data[
+                        "is_building_area_MF_dormitory_or_hotel_b"
+                    ]
+
+                    space_lighting_or_vent_space_type_is_defined = not (
+                        space_b.get("lighting_space_type") is None
+                        and space_b.get("ventilation_space_type_b") is None
```

Comment:
This should be `ventilaton_space_type`

Reply:
Also, this is a little confusing with not and is None
How about:
```python
space_lighting_or_vent_space_type_is_defined = space_b.get("lighting_space_type") or space_b.get("ventilation_space_type_b")
```

Reply:
Good point! Addressed.